crates/runtime/src/http/routes.rs

```diff
@@ -46,6 +46,7 @@ pub(crate) fn routes(
         .route("/v1/status", get(v1::status::get))
         .route("/v1/datasets", get(v1::datasets::get))
         .route("/v1/datasets/:name/refresh", post(v1::datasets::refresh))
```

Comment:
Should we move `/refresh` to `/acceleration/refresh`?

Reply:
Updated, thank you!